src/gui/uithememanager.h

```diff
@@ -31,6 +31,7 @@
 
 #include <QObject>
 
+class ApplicationPalette;
```

Comment:
Qt class declarations should come before our local ones, so move this one line lower.

Reply:
done.